changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.13.2
+
+* Fix: Don't let Range header persist into other requests (#2195)
```

Comment:
nit: I don't think we need "Fix" in front of these, they're all fixes :) 

Since I need to rebase anyways I'll remove them and then merge.